### PR TITLE
Feature/app manual

### PR DESCRIPTION
**Pull request by janaRicarda**
Add manualData to lib folder,
add app-manual page,
add component `Manual`and render it onn app-manual page.

### CODE REVIEW

File: components/Manual.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    border-bottom: 2px dotted var(--button-background);
+    padding-right: 2rem;
+  }
+`;
+
+export default function Manual() {
+  return (
+    <>
+      <StyledSection>
+        <StyledWelcome>Welcome to the What a feeling app!</StyledWelcome>
```
Comment (F-Kirchhoff) [inline components/Manual.js:107]: I would suggest to highlight the app name here:
```suggestion
        <StyledWelcome>Welcome to the <b>What a Feeling</b> App!</StyledWelcome>
```

File: components/Manual.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+            <StyledOl>
+              {answers.map((answer) => (
+                <StyledListItem key={answer}>{answer}</StyledListItem>
+              ))}
+            </StyledOl>
+          </StyledDetails>
+        ))}
+
+        <StyledText>
+          Remember, the What a feeling app is here to assist you in
```
Comment (F-Kirchhoff) [inline components/Manual.js:127]: here you can do the same to improve readability.

**Review by F-Kirchhoff — APPROVED** [review]
LGTM, just one small suggestion about the App name in your manual text. Feel free to implement or to ignore.

**Comment by janaRicarda** [comment]
Thank you for your review and your recommendation! We'll implement your suggestion.

**Pull request merged**